modules/vaos/app/services/vaos/v1/fhir_service.rb

```diff
@@ -31,6 +31,14 @@ def read(resource_type, id)
         perform(:get, "#{resource_type}/#{id}", nil)
       end
 
+      def search(resource_type, query_string)
```

Comment:
Looks like there were some changes to the methods in this file here, might want to wait until this PR gets merged https://github.com/department-of-veterans-affairs/vets-api/pull/4294/files#diff-610e92fc063ec6e9582cc8faec6dd928R9